src/functionalTest/java/uk/gov/hmcts/reform/draftstore/data/OldDraftsCleanupTest.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.draftstore.data;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.reform.draftstore.endpoint.v3.helpers.SampleData;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.util.function.Consumer;
+
+import static java.time.Instant.now;
+import static java.time.temporal.ChronoUnit.DAYS;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
+public class OldDraftsCleanupTest {
+
+    @Autowired
+    private NamedParameterJdbcTemplate jdbcTemplate;
+    private Instant now = now();
+
+    @Test
+    public void should_remove_stale_drafts() throws Exception {
+        DraftStoreDAO nowRepo = repoAtTime(now);
+
+        int d1 = nowRepo.insert("", "", SampleData.createDraft(5));
+        int d2 = nowRepo.insert("", "", SampleData.createDraft(15));
```

Comment:
Not that important, but those values could be 10 and 11 - this way we would have the behaviour around the threshold fully checked.